src/System.Memory/tests/ReadOnlyBuffer/ReadOnlySequenceTests.Default.cs

```diff
@@ -122,6 +122,33 @@ public void Default_SliceNegativeLength()
             Assert.Throws<ArgumentOutOfRangeException>("length", () => buffer.Slice(buffer.Start, -1L));
         }
 
+        [Fact]
+        public void Slice_DefaultSequencePosition()
+        {
+            var firstSegment = new BufferSegment<byte>(new byte[4]);
+            BufferSegment<byte> secondSegment = firstSegment.Append(new byte[4]);
+
+            var sequence = new ReadOnlySequence<byte>(firstSegment, 0, secondSegment, firstSegment.Memory.Length);
+            ReadOnlySequence<byte> slicedSequence = sequence.Slice(default(SequencePosition));
+            Assert.Equal(sequence, slicedSequence);
+
+            // Slice(default, default) should return an empty sequence
+            slicedSequence = sequence.Slice(default(SequencePosition), default(SequencePosition));
+            Assert.Equal(0, slicedSequence.Length);
+
+            // Slice(x, default) returns empty if x = 0. Otherwise throws
+            sequence = sequence.Slice(2);
```

Comment:
This should cover both the Slice(int, SequencePosition) and the Slice(SequencePosition, int) cases

Reply:
How so? I would feel more confident if we had separate test cases for both overloads, explicitly.

Reply:
We discussed this offline, and it makes sense to me now.